bodhi/tests/server/consumers/test_greenwave.py

```diff
@@ -0,0 +1,104 @@
+# Copyright © 2016-2019 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This test suite contains tests for the bodhi.server.consumers.greenwave module."""
+
+from unittest import mock
+import unittest
+
+from fedora_messaging.api import Message
```

Comment:
For PEP-8, there should be an empty line between third party imports and Bodhi imports:

```suggestion
from fedora_messaging.api import Message

```